pkg/util/node/BUILD

```diff
@@ -11,15 +11,18 @@ go_library(
     srcs = ["node.go"],
     importpath = "k8s.io/kubernetes/pkg/util/node",
     deps = [
+        "//pkg/features:go_default_library",
```

Comment:
low pri - Would be great if we move all the generated stuff into a separate commit, making it easier to skip in review